src/ck_game.c

```diff
@@ -836,6 +836,13 @@ void CK_TryAgainMenu()
 			{
 				sel = 1;
 			}
+
+			if (IN_GetLastScan() == in_kbdControls.quickLoad)
+			{
+				// quickload, but fall back to map screen on failure
+				ck_gameState.currentLevel = 0;
+				return US_QuickLoad();
+			}
 		} /* while */
 	}
 }
```

Comment:
Need a "return" statement here.

Reply:
Argh, of course! :/
Maybe the Makefile needs a `-Wall` by default so stuff like this is harder to miss ...